src/test/java/uk/gov/dhsc/htbhf/BrowserStackConfiguration.java

```diff
@@ -0,0 +1,43 @@
+package uk.gov.dhsc.htbhf;
+
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.remote.DesiredCapabilities;
+import org.openqa.selenium.remote.RemoteWebDriver;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.Profile;
+import org.springframework.context.annotation.PropertySource;
+import uk.gov.dhsc.htbhf.utils.NoopWireMockManager;
+import uk.gov.dhsc.htbhf.utils.WireMockManager;
+
+import java.net.MalformedURLException;
+import java.net.URL;
+
+@Configuration
+@PropertySource("classpath:application-browserstack.properties")
+@Profile("browserstack")
+public class BrowserStackConfiguration {
+
+    @Bean(destroyMethod = "close")
+    public WebDriver browserStackWebDriver() throws MalformedURLException {
+        //TODO MRS 2019-08-14: Import via config all these values
+        String BROWSER_STACK_USER = "dhschtbhfsupport1";
+        String BROWSER_STACK_KEY = "RRxF4gChswf8qzC1nh8s";
```

Comment:
We probably shouldn't have the browserstack key in source code

Reply:
Done